framework/include/constraints/ConstraintWarehouse.h

```diff
@@ -45,6 +46,7 @@ class ConstraintWarehouse : public MooseObjectWarehouse<Constraint>
    */
   const std::vector<MooseSharedPointer<NodalConstraint> > & getActiveNodalConstraints() const;
   const std::vector<MooseSharedPointer<FaceFaceConstraint> > & getActiveFaceFaceConstraints(const std::string & interface) const;
+  const std::vector<MooseSharedPointer<ElemElemConstraint> > & getActiveElemElemConstraints(unsigned int interface_id) const;
```

Comment:
How do you specify `interface_id` in the input file? Or is that info sitting in the mesh file.
We might need a typedef for it - that would make the code a bit more legible...